web/src/store/ApiStore.ts

```diff
@@ -87,13 +87,14 @@ export const useApiStore = defineStore("api", () => {
     } */
     response = await auth0.getAccessTokenSilently().then(async (token) => {
       return await SecureAPICall(method, token)
```

Comment:
Any API access should be "secure by default"; this API setup is **non-secure** by default. As such, I would pull in the `http-client` pattern from https://github.com/icefoganalytics/wrap/blob/main/web/src/api/http-client.ts, and start propagating it throughout the codebase as you need it.